src/QMCWaveFunctions/Fermion/MatrixDelayedUpdateCUDA.h

```diff
@@ -293,32 +293,32 @@ class MatrixDelayedUpdateCUDA
     if (n_accepted == 0)
       return;
 
-    auto& hstream              = engine_leader.cuda_handles_->hstream;
-    auto& updateRow_buffer_H2D = engine_leader.mw_mem_->updateRow_buffer_H2D;
-    auto& mw_temp              = engine_leader.mw_mem_->mw_temp;
-    auto& mw_rcopy             = engine_leader.mw_mem_->mw_rcopy;
-    auto& cone_vec             = engine_leader.mw_mem_->cone_vec;
-    auto& czero_vec            = engine_leader.mw_mem_->czero_vec;
-    const int norb             = engine_leader.get_ref_psiMinv().rows();
-    const int lda              = engine_leader.get_ref_psiMinv().cols();
+    auto& hstream               = engine_leader.cuda_handles_->hstream;
+    auto& updateRow_buffer_H2D  = engine_leader.mw_mem_->updateRow_buffer_H2D;
+    auto& mw_temp               = engine_leader.mw_mem_->mw_temp;
+    auto& mw_rcopy              = engine_leader.mw_mem_->mw_rcopy;
+    auto& cone_vec              = engine_leader.mw_mem_->cone_vec;
+    auto& czero_vec             = engine_leader.mw_mem_->czero_vec;
+    const int norb              = engine_leader.get_ref_psiMinv().rows();
+    const int lda               = engine_leader.get_ref_psiMinv().cols();
+    const int nw                = engines.size();
+    const size_t phi_vgl_stride = nw * norb;
     mw_temp.resize(norb * n_accepted);
     mw_rcopy.resize(norb * n_accepted);
-    updateRow_buffer_H2D.resize((sizeof(Value*) * 8 + sizeof(Value)) * n_accepted);
+    updateRow_buffer_H2D.resize((sizeof(Value*) * 6 + sizeof(Value)) * n_accepted);
```

Comment:
What does the "6" mean and why was it changed from "8"? 
The "6" gets reused in a few more places, it might be worth making it a named variable.